src/jobs/js_push_build_to_bucket_job.yml

```diff
@@ -6,6 +6,13 @@ docker:
   - image: gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine
 
 steps:
+  - when:
+      condition:
+        equal: [ true, << parameters.do_checkout >> ]
+      steps:
+        # in auth0 build files are committed
+        - checkout
+  - checkout
```

Comment:
to me this looks like checking out with an extra step :) what do you want to achieve?

Reply:
🤦